### PR TITLE
Tech docu

### PR DESCRIPTION
**Pull request by Pratichhya**

### CODE REVIEW

**Comment by jdries** [comment]
Hi @Pratichhya the pull request contains a _site directory, which should not be checked into git.
Is there a way to remove it from history, or do we need a new PR for that?

File: Roadmap.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 The [Service description and evolution document](/_docs/CDSE-SDE-TSY_Service%20Description%20and%20Evolution.pdf ':ignore') provides a comprehensive view of all data products and services available in the frame of the Copernicus Data Space Ecosystem, both the “Copernicus Free Services” as well as the services operated by third parties. 
 
+test
```
Comment (jdries) [inline Roadmap.md:6]: remove?

Comment (Pratichhya) [reply Roadmap.md:6]: yes seems to be there mistakely

**Comment by Pratichhya** [comment]
> 

I tried to avoid it by including it in gitignore but it seems like anyway it was there in PR. I am not sure if there is a way to remove it in history or if I should create a new PR.

**Comment by jdries** [comment]
In general, it's best to check upon commit what files are included. The gitignore file should help, but there's always a chance that some files still end up in your commit unwanted.
It is indeed possible to filter certain files from history (you can find approaches online) or indeed just create a new branch with entirely new commits.

**Comment by Pratichhya** [comment]
> In general, it's best to check upon commit what files are included. The gitignore file should help, but there's always a chance that some files still end up in your commit unwanted. It is indeed possible to filter certain files from history (you can find approaches online) or indeed just create a new branch with entirely new commits.

Ahh ok sure I will do that

**Pull request closed without merging**